Cargo.toml

```
@@ -43,7 +43,7 @@ debug = true
 [dependencies]
 fixedbitset = { version = "0.5.7", default-features = false }
 indexmap = { version = "2.5.0", default-features = false }
-hashbrown = { version = "^0.15.0" }
+hashbrown = { version = "^0.15.0", default-features = false, features = ["default-hasher"] }
```

```suggestion
hashbrown = { version = "^0.15.0", default-features = false, features = ["default-hasher", "inline-more"] }
```

Thanks for the PR! I think keeping a potential performance improvement would be the right decision.